### ISSUE TITLE
2nd Edition

### ISSUE THREAD

**Issue by sirjorj** [sirjorj/xhud#22]
So... what to do about this...

I will probably make libxwing2 as a separate project, as it looks like there are a lot of changes to existing cards.  This way, the old data can still happily exist if anyone wants it and not bloat the new library.

I will probably do the same with xhud.  If I combine them, a) the user will have to decide which dataset to use, and b) the old data will just be making the file bigger.  Separating them will keep xhud2 limited to new things and allow xhud to keep existing for anyone who wants to play the old way.

It is probably to early to start building libxwing2, as a lot of the new mechanics have not been explained, but that probably won't stop me. :)

**Comment by urbknu** [comment]
I answered your questions on xwing-font as if it where xhud ;)

**Comment by sirjorj** [comment]
Heh.  I saw your post over there!  I'm probably going to make xhud2/lilbxwing2 so the old versions can still exist.

**Comment by urbknu** [comment]
Need to figure out how we can help out in this endeavour - without having to learn C++ ;) Seems to me there will be some major changes ahead.

**Comment by sirjorj** [comment]
My goal is to have libxwing2/xhud2 ready to go when the game launches - or at least as much as possible.  Data entry will be a big part, so those conversion kits will have a lot of stuff to enter - lots of proofreading and testing!

**Comment by urbknu** [comment]
Sign me up! Do you use xwing-data as master?

**Comment by sirjorj** [comment]
No, i enter it manually into the libxwing project.  Last night I finally put a good start into libxwing2.  Hopefully this weekend I can get the xwing.cgi website to show the new data.
If you want, you can drop me an email at github at sirjorj dot com so we can work out some details.
